yt_dlp/extractor/eurosport.py

```diff
@@ -3,7 +3,7 @@
 
 
 class EurosportIE(InfoExtractor):
-    _VALID_URL = r'https?://www\.eurosport\.com/\w+/(?:[\w-]+/[\d-]+/)?[\w-]+_(?P<id>vid\d+)'
+    _VALID_URL = r'https?://(?:(?:www\.)?eurosport\.(?:com(?:\.tr)?|de|dk|es|fr|hu|it|nl|no|ro)|eurosport.tvn24.pl)/[\w-]+/(?:[\w-]+/[\d-]+/)?[\w.-]+_(?P<id>vid\d+)'
```

Comment:
Also:
`https://www.tntsports.co.uk/`
`https://espanol.eurosport.com/`
Just found out that these domain names can be retrieved from the 'CHANGE VERSION' list at the bottom of the page.
![image](https://github.com/user-attachments/assets/3620679c-f312-4226-9da1-fc11a70010d5)

Reply:
The redirect of tntsports.co.uk to eurosport.com is a geo-block. Though, even with a VPN I get 403s and 451s on the media endpoints, most likely due to using a VPN. The one video I was able to access was DRM protected.
For now, I would leave this out until someone from the UK can confirm what works and what doesn't.

I'll add `espanol.eurosport.com`. Thanks!

Reply:
There are too many domain names now. I suggest creating a list of domains like in the VidflexIE pr.

Reply:
IMO a list isn't necessary, but a multiline regex pattern might be nice to break up the long line

Reply:
But the current regex matches `espanol.eurosport.de`, which doesn't seem to work. Does that matter?

Reply:
Yep, I'm aware. I think it is fine to keep the regex a bit simpler.

Reply:
> I think it is fine to keep the regex a bit simpler.

agreed. it's only a problem if the user manually constructs a bogus url